generation/WinSDK/enums.json

```
@@ -34862,6 +34862,52 @@
       }
     ]
   },
+  {
+    "name": "CDN_CONSTANTS",
```

Perhaps better named this? (https://learn.microsoft.com/windows/win32/dlgbox/common-dialog-box-notifications)

```suggestion
    "name": "COMMON_DIALOG_NOTIFICATIONS",
```

---

Should be singular not plural if this isn't a flags enum.